### ISSUE TITLE
relu at the end of encoder

### ISSUE THREAD

**Issue by Bemfool** [pixelite1201/pytorch_coma#21]
After training, I noticed that all latent codes are not less than 0, which might be caused by F.relu at the end encoder. I think it's not neccessary and harmful for variational CoMA. So I replace it with leakyReLU for mean and Sigmoid for std, which works.
